metsim/io.py

```diff
@@ -35,12 +35,12 @@ def read_ascii(fpath, n_days=-1):
         for line in csv.reader(f, delimiter='\t'):
             print(line)
             break
+    # TODO: FIXME: Finish this
+    return Forcing(None, None)
 
 
-def read_binary(fpath, n_days=-1):
-    """
-    TODO
-    """
+def read_binary(fpath: str, n_days=-1) -> Forcing:
+    """ Reads a binary forcing file (VIC 4 format) """
```

Comment:
nit: the pep8 docs don't have a space around the triple quotes in docstrings so I don't think they are necessary.